utilities/rali/rali_unittests/rali_unittests.cpp

```diff
@@ -410,15 +478,20 @@ int test(int test_case, const char* path, const char* outName, int rgb, int gpu,
             image1 = raliResizeCropMirrorFixed(handle, image0, 100, 100, true, 50, 50, 0);
         }
             break;
-	
+        case 54: {
+             std::cout << ">>>>>>> Running " << "raliSSDRandomCrop" << std::endl;
+             image1 = raliSSDRandomCrop(handle, input1, true);
+        }
+        break;
+
 	default:
             std::cout << "Not a valid option! Exiting!\n";
             return -1;
     }
 
     // Calling the API to verify and build the augmentation graph
     raliVerify(handle);
-
+    image1 = nullptr;
```

Comment:
Codacy found an issue: [Variable 'image1' is reassigned a value before the old one has been used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)